t/ui/13-admin.t

```diff
@@ -449,9 +449,7 @@ subtest 'job property editor' => sub() {
             undef, 'group properties save button is enabled');
         $driver->find_element_by_id('editor-carry-over-bugrefs')->click();
         $driver->find_element('#properties p.buttons button.btn-primary')->click();
-        # ensure there is no race condition, even though the page is reloaded
-        wait_for_ajax;
-
+        wait_for_ajax(msg => 'ensure there is no race condition, even though the page is reloaded');
```

Comment:
I'm wondering why we have this `wait_for_ajax` here but I suppose let's not touch a running system :-)